test/powershell/Language/Scripting/NativeExecution/NativeCommandProcessor.Tests.ps1

```diff
@@ -42,20 +42,20 @@ Describe "Native Command Processor" -tags "Feature" {
     It "Should kill native process tree" -Pending {
 
         # make sure no test processes are running
-        # on Linux, the Process class truncates the name so filter using Where-Object
-        Get-Process | Where-Object {$_.Name -like 'createchildproc*'} | Stop-Process
+        Get-Process testexe -ErrorAction SilentlyContinue | Stop-Process
```

Comment:
Docs say that by default, Stop-Process prompts for confirmation - we should add "-Force".